### DISCUSSION TITLE
Doubt about pvc access mode

### DISCUSSION THREAD

**Discussion by felipejareta** [minio/minio#16506] [Q&A] [answered]
Hi guys.

I have a doubt about PVC access mode. I am using Openshift and my Minio is there. I would like to get more availability with Minio and turn my Access Mode RWO settings to ReadWriteMany. So my question is...Is there any limitation for Minio to use this kind of access mode or recommendation to don't use this option?

Regards.

**Comment by ravindk89** [comment, marked as answer]
https://min.io/docs/minio/kubernetes/upstream/operations/install-deploy-manage/deploy-minio-tenant.html#persistent-volumes

We support `ReadWriteOnce` access modes on underlying PVC only. 

Under what circumstances do you expect `ReadWriteMany` to improve availability with MinIO?

**Reply by felipejareta** [reply]
Right.

For example when a Openshift node become not ready, my Minio pod can't up in other node because the PVC minio is still store in node not ready.
I would like to avoid this and give more availability to the Minio.

**Reply by ravindk89** [reply]
That's not giving availability to MinIO - the pods should use storage local to the node. Or if this is a networked provider, then you are relying on the availability of that provider. `readWriteMany` isn't the solution.

If you have 4 pods on 4 nodes, but they are all using storage on one node, `ReadWriteMany` isn't solving any problems. You at most have a false sense of availability because the four nodes can come up, but your single node where storage is located is your failure point.

The recommended route is for each node to have node-local storage. If you use networked PVCs, then the network provider *must* support `readWriteOnce` to ensure proper treatment of MinIO's access patterns.

`readWriteMany` is not supported. You can use and experiment with it at your own risk.

**Reply by felipejareta** [reply]
Got it.

Thank you for your answer.
